ginga/util/io_fits.py

```diff
@@ -246,18 +250,48 @@ def get_hdu_type(self, hdu):
 
         return None
 
-    def load_hdu(self, hdu, dstobj=None, **kwargs):
+    def load_hdu(self, hdu, dstobj=None, fobj=None, naxispath=None,
+                 save_primary_header=False, **kwargs):
```

Comment:
Might also want to mention this in the change log. Addition might still break API if someone has been calling the keywords as arguments.